src/module-2/components/stylesheets/results.css

```diff
@@ -0,0 +1,68 @@
+.results{
+  display: flex;
+  flex-direction: column;
+  gap:15rem;
+}
+.results .caption button{
+  background:#6F64F8;
+}
+.results .caption button img{
```

Comment:
try to avoid large nesting hierarchies

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)